Magento_Customer/templates/address/edit.phtml

```diff
@@ -432,9 +432,9 @@
                 "regionInputId": "#region",
                 "postcodeId": "#zip",
                 "form": "#form-validate",
-                "regionJson": <?= $this->helper(\Magento\Directory\Helper\Data::class)->getRegionJson() ?>,
+                "regionJson": <?= $block->escapeJs($this->helper(\Magento\Directory\Helper\Data::class)->getRegionJson()) ?>,
                 "defaultRegion": "<?= (int) $block->getRegionId() ?>",
-                "countriesWithOptionalZip": <?= $this->helper(\Magento\Directory\Helper\Data::class)->getCountriesWithOptionalZip(true) ?>
+                "countriesWithOptionalZip": <?= $block->escapeJs($this->helper(\Magento\Directory\Helper\Data::class)->getCountriesWithOptionalZip(true)) ?>
```

Comment:
for regionJs and countriesWithOptionalZip no escape in this case isntead of escapeJs (the way it is in original file)